<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QCheckBox" name="use_db_checkBox">
     <property name="text">
      <string>Use Diablo Database</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="db_info_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QGroupBox" name="uid_groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="TidyComboBox" name="login_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="TidyComboBox" name="password_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="windows_auth_checkBox">
        <property name="text">
         <string>Use Windows Authentication</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Server name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Database name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="TidyComboBox" name="server_name_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="TidyComboBox" name="db_name_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="use_wsi_checkBox">
     <property name="text">
      <string>Use Windshear Data Directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="ok_pushButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="cancel_pushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="wsi_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TidyComboBox" name="wsi_dir_comboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="wsi_browse_pushButton">
        <property name="text">
         <string>Browse Directories</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TidyComboBox</class>
   <extends>QComboBox</extends>
   <header>tidy_combo_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
